.migrations/V20250122141100__add_cluster_routing.sql

```diff
@@ -0,0 +1,39 @@
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'tier') THEN
+CREATE TYPE tier AS ENUM ('Tier0', 'Tier1', 'Tier2', 'Tier3', 'Tier4');
```

Comment:
Tier 0 doesn't exist, yeah?

```suggestion
CREATE TYPE tier AS ENUM ('Tier1', 'Tier2', 'Tier3', 'Tier4');
```